src/meta/src/manager/notification.rs

```diff
@@ -222,6 +227,7 @@ impl NotificationManagerCore {
             WorkerType::Frontend => &mut self.frontend_senders,
             WorkerType::ComputeNode => &mut self.compute_senders,
             WorkerType::Compactor => &mut self.compactor_senders,
+            WorkerType::RiseCtl => &mut self.risectl_senders,
```

Comment:
>  // We don't care about any update so far.

Risectl doesn't care about any update so we don't need to send notification to it? If it just needs an init snapshot, can we get it by using another RPC instead of `Subscribe`?